### ISSUE TITLE
GGMAX Enhancement - Character Creator

### ISSUE THREAD

**Issue by bond1mjb** [TheGameCreators/GameGuruRepo#5530]
No rush, but a few requests for enhancements to the CC.  

1.  Firstly a minor cosmetic request.   At the moment, all custom CC folders automatically default to the zombie locker room.  It would be preferable if it they could use one of the other two "normal" locker rooms instead.  Or ideally, have a way to specify which locker room is used (male,female, zombie).

2. At the moment, characters from custom CC folders automatically default to use the MALE animation sets.  It would be ideal if they would by default use the animation sets stored within the custom parts folder.  The same way the MALE, FEMALE, and ZOMBIES do.  The animation sets can be manually overridden in the behavior itself, but that would lead to confusion for end-users if they were required to do this each time when setting the behavior.
![custom](https://github.com/TheGameCreators/GameGuruRepo/assets/87768878/96b03d0a-3333-4b36-a855-73b6512496f7)

**Comment by LeeBamberTGC** [comment]
@bond1mjb Retagging this ias a bug as I agree the character animations chosen in the CCP system should match the style selected and use the right files.  Would you say the male locker room is the more normal one?

**Comment by bond1mjb** [comment]
@LeeBamberTGC That's great!  Yes I think the male locker room would be a good choice.

**Comment by Necrym59** [comment]
Just a side note:
The 'locker room' is just a dbo isnt it, can we have a selection box so people can point to/supply their own locker room file as an option?

**Comment by bond1mjb** [comment]
@Necrym59 Yes that's right, the locker room is just a DBO model and associated texture.  I suppose it depends how much work Lee wants to put into this.  But I'd agree it would be neat if each custom folder could also optionally "point" to a custom locker room for that extra visual flair:)

**Comment by LeeBamberTGC** [comment]
NOTE: If ANYONE can produce a DBO+Texture that aligns and works with the CCP "room" I will spend the time to add the feature to change the room to a custom user defined one instead of the stock one.  Hopefully someone can create a FALLOUT 4 Power Armour Garage scene, that would be my character creator area ;)

**Comment by mav3r1ck1981** [comment]
@LeeBamberTGC on it

**Comment by Necrym59** [comment]
@LeeBamberTGC 
Two gracious and talented people namly, Mav3r1ck1981 and C4ever08 have kindly supplied two extra CCP rooms, - a wasteland style garage and a futuristic style room. 
![wasteland](https://github.com/TheGameCreators/GameGuruRepo/assets/95015657/11526999-5024-4f09-ad47-4dcf9cfbabd4)
![futuristic](https://github.com/TheGameCreators/GameGuruRepo/assets/95015657/5c788fcd-cdb4-4da2-9f19-07f8ea5c2f1a)
I will email you the download links. 
As the future for the CC could eventually be used as an in-game player character creator having different backdrop rooms capability would be a nice touch.

**Comment by LeeBamberTGC** [comment]
@Necrym59 These are awesome, what a neat idea to enable your own custom room for making your characters, and maybe down the road different workshop rooms for importing objects, making weapons, crafting composite objects, etc :)

**Comment by LeeBamberTGC** [comment]
I have updated the DEVTEST branch with the new implementation for character creator plus custom rooms, keeping the defaults for the first four types as before, but now fully expandable and customizable with the two extra ones. The system also remembers the room references if you leave MAX and relaunch, and new character type categories can include a roompref.txt file that will auto-set the preferred room to match the character type content.

I have also fixed the issue of custom character types now using their own default animation file animations by default  Again this will be in the DEVTEST branch so worth testing before we close this one.
